spec/game_spec.rb

```diff
@@ -0,0 +1,44 @@
+require './game'
+require './author'
+require 'date'
+
+describe Game do
+  let(:author) { Author.new('Hidetaka', 'Miyazaki') }
+  let(:today) { Date.today.to_s }
+  let(:multiplayer) { true }
+  let(:over_decade) { Date.parse(today).prev_year(11).to_s }
+
+  context 'when receiving valid parameters' do
+    let(:game) { Game.new(author, today, multiplayer, over_decade) }
+
```

Comment:
- [ ] One of the project requirements is that all public methods in all classes should have unit tests. Please write unit tests for the `to_hash` method in your `Game` class 👍